id: PYSEC-0000-CVE-2022-34265
details: An issue was discovered in Django 3.2 before 3.2.14 and 4.0 before 4.0.6.
  The Trunc() and Extract() database functions are subject to SQL injection if untrusted
  data is used as a kind/lookup_name value. Applications that constrain the lookup
  name and kind choice to a known safe list are unaffected.
affected:
- package:
    name: django
    ecosystem: PyPI
    purl: pkg:pypi/django
  ranges:
  - type: ECOSYSTEM
    events:
    - introduced: "3.2"
    - fixed: 3.2.14
    - introduced: "4.0"
    - fixed: 4.0.6
references:
- type: ARTICLE
  url: https://www.djangoproject.com/weblog/2022/jul/04/security-releases/
- type: WEB
  url: https://docs.djangoproject.com/en/4.0/releases/security/
- type: WEB
  url: https://groups.google.com/forum/#!forum/django-announce
aliases:
- CVE-2022-34265
modified: "2022-07-04T16:15:00Z"
published: "2022-07-04T16:15:00Z"
